website/content/partials/known-issues/database-skip-static-role-rotation.mdx

```diff
@@ -14,4 +14,6 @@ their passwords rotated when Vault is restarted. The password rotation will not
 occur on restart if the static role has already had its password rotated.
 
 #### Workaround
-Currently, there is no workaround except to avoid a restart of the backend.
+This issue has been fixed in Vault versions 1.18.6 and 1.19.0. If you are experiencing this issue, upgrade to one of these versions or newer.
+
+For earlier versions, there is no workaround except to avoid a restart of the backend.
```

Comment:
```suggestion
Upgrade to Vault version 1.18.6+ and 1.19.1+. For earlier versions of Vault, the only
workaround avoiding restarts of the backend.
```

Style guide correction: Write in active voice, avoid "this" as a pronoun

Also, how does upgrading to 1.19.0 fix the problem if 1.19.0 is the affected version? Wouldn't they need to upgrade to 1.19.1?